packages/redux/src/themes/selectors.ts

```diff
@@ -0,0 +1,49 @@
+import { getError, getIsLoading, getResult } from './reducer.js';
+import type { StoreState } from '../types/index.js';
+import type { Theme } from '@farfetch/blackout-client';
+import type { ThemeState } from './types/index.js';
+
+/**
+ * Checks if styleguide theme has error.
+ *
+ * @param state - Application state.
+ * @param code  - The theme identifier.
+ *
+ * @returns Staff member error.
```

Comment:
```suggestion
 * @returns Theme error.
```

Check other `@returns` tags in this file as they have the same problem.

Reply:
Done